tweets/views.py

```diff
@@ -12,4 +12,30 @@ def createTweet(request):
 
 def showTweet(request,id):
   tweet_details = get_object_or_404(Tweet, id=id)
-  return render(request, 'tweets/show.html',{'tweet_details':tweet_details})
\ No newline at end of file
+  return render(request, 'tweets/show.html',{'tweet_details':tweet_details})
+
+def editTweet(request,id):
+  tweet_details = get_object_or_404(Tweet, id=id)
```

Comment:
Why not put request.user in query and only find the ones created by user

Reply:
This can be done

Reply:
I've done to 
```python
tweet_details = get_object_or_404(Tweet, id=id, user_id=request.user.id)
```